app/presenters/siteimprove/policy_issues_presenter.rb

```diff
@@ -0,0 +1,53 @@
+require "govspeak"
+
+module Siteimprove
+  class PolicyIssuesPresenter
+    attr_reader :summary_info
+
+    def initialize(policy_issues, summary_info)
+      @policy_issues = policy_issues
+      @summary_info = summary_info
+    end
+
+    def any?
+      @policy_issues.any?
+    end
+
+    def issue_list(type)
+      @policy_issues.select { |issue| type == :all || policy_type?(issue.policy_name) == type }.map do |issue|
+        {
+          policy_category: issue.policy_category,
+          gds_policy_id: gds_policy_id(issue.policy_name),
+          policy_short_description: policy_short_description(issue.policy_name),
+          policy_priority: issue.policy_priority,
+          policy_description: policy_description(issue),
+          policy_direct_link: "#{link}#issue/#{issue.id}",
+        }
+      end
+    end
+
+    def link
+      summary_info._siteimprove.policy.page_report.href
+    end
+
+    def policy_type?(policy_name)
```

Comment:
I think the `?` in this method's name should be dropped, as it doesn't return a boolean

(I assume it used to return a boolean in your spike and it just hasn't been renamed since then?)